Automattic/wp-calypso
# Update/plan overhaul revert - fix spacing between notifications

#### Proposed Changes

Addresses 820-gh-Automattic/martech

* Branch `update/plan-overhaul-revert-fix-spacing` is cut from `update/plan-overhaul-revert`
* Update `classNames` adding is-wpcom conditionally
* Use `is-wpcom` `className` to fix the spacing betweem the `Credits Notice` and the `Non-renewable Plan Notice` 

#### Testing Instructions

* Add (or be sure to have) WPCom credits in your WPCom account [https://wordpress.com/wp-admin/network/admin.php?page=store-admin](https://wordpress.com/wp-admin/network/admin.php?page=store-admin
* Create (or select) a `Pro plan` or `Starter plan` site
* Visit http://calypso.localhost:3000/plans/<YOUR SITE>
* Verify the gap between the Credits Notice and the Non-renewable Notice is resolved.

#### Before / After

##### Before
<img width="1478" alt="Screenshot 2022-07-18 at 15 57 15" src="https://user-images.githubusercontent.com/5706607/179565077-35a5747f-73f7-4401-9d78-94affa629f4d.png">

##### After
<img width="1459" alt="Screenshot 2022-07-18 at 19 06 36" src="https://user-images.githubusercontent.com/5706607/179565122-e17b2df4-18bc-4cd7-b65b-390da7ad4d8f.png">
